docs/_includes/variation-content.html

```diff
@@ -177,7 +177,7 @@ <h4 class="variation-code-snippet-title">HTML code snippet</h4>
 
                 <!-- Jinja snippet -->
                 {% if jinja_snippet != "" %}
-                <section class="govuk-tabs__panel govuk-tabs__panel--hidden" id="{{ variation_slug }}-jinja-code-snippet-{{ forloop.index }}">
+                <section class="m-tabs__panel m-tabs__panel--hidden" id="{{ variation_slug }}-jinja-code-snippet-{{ forloop.index }}">
```

Comment:
Here they're putting a modifier on the `panel` element. We don't put modifiers on elements much or at all, so it's discouraged, but if it makes it easier to migrate this over it's fine. However, I think we'll want to use `u-hidden` (utility class) to handle the hide/show as opposed to a modifier.